fili-core/src/main/java/com/yahoo/bard/webservice/data/config/table/BaseTableLoader.java

```diff
@@ -310,7 +295,8 @@ protected PhysicalTable buildPhysicalTable(
                 definition.getName(),
                 definition.getGrain(),
                 columns,
-                definition.getLogicalToPhysicalNames()
+                definition.getLogicalToPhysicalNames(),
+                metadataService
```

Comment:
Well... the sub-class should simply call `super()` with their own `metadataService`?